tests/foreman/ui/test_bookmarks.py

```diff
@@ -1,25 +1,79 @@
 """Test classes for Bookmark tests"""
 # -*- encoding: utf-8 -*-
-from robottelo.constants import BOOKMARK_ENTITIES
-from robottelo.decorators import stubbed, tier1, tier2
+from fauxfactory import gen_string
+from nailgun import entities
+from random import choice
+from robottelo.constants import BOOKMARK_ENTITIES, STRING_TYPES
+from robottelo.decorators import (
+    bz_bug_is_open,
+    skip_if_bug_open,
+    stubbed,
+    tier1,
+    tier2,
+)
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.session import Session
 
 
-def create_entity(entity):
-    """Take entity name and call nailgun methods to create a random instance
-
-    :param entity: Nailgun entity class to be processed
+def gen_random_string(length=10):
```

Comment:
just a suggestion. Can't we re-use our `datafactory` method and take only one, random element from the list? However, it will take more CPU instructions :)